### ISSUE TITLE
Update FAQ now that Space ROS has released

### ISSUE THREAD

**Issue by Bckempa** [space-ros/spaceros.org#27]
The FAQ talks about an upcoming release in 2022. This obviously needs to be update.

**Comment by elaske** [comment]
Is this just about the "When is Space ROS available?" question?

Or, do we need to add / update other information? For example:
* Roadmap (if any)
* Steering Committee information (how often, who to contact, etc.)
* Update how to get involved - it doesn't even link to GitHub right now, let alone maybe a way to guide toward a POC for technical engagement.
* ROS Discourse?

**Comment by Bckempa** [comment]
Yes to all that but those other things should technically get their own issues 🙃

**Comment by elaske** [comment]
Arguably this issue needs better scoping then 🤣 but we can split off
issues for every FAQ update if we want. This issue doesn't comply with the
new issue types and required fields right now either.
